### PR TITLE
feat: Adding docker-compose.yml

### PR DESCRIPTION
**Pull request by Shrikant1212**
# Description
Configured docker-compose support for deployment

## Type of change

Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?
We can run razor-go with command `docker-compose up -d`
# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ashish10677** [comment]
@Shrikant1212 can you update the README as well?

File: README.md [hunk 1]
```diff
@@ -9,13 +9,15 @@ Official node for running stakers in Golang.
...
   docker run -d \
   -it \
   --name razor-go \
   -v "$(echo $HOME)"/.razor:/root/.razor \
   razornetwork/razor-go
 ```
+
 Note that we are leveraging docker bind-mounts to mount `.razor` directory so that we have a shared mount of `.razor` directory between the host and the container. The `.razor` directory holds keys to the addresses that we use in `razor-go`, along with logs and config. We do this to persist data in the host machine, otherwise you would lose your keys once you delete the container.
 
 You need to set a provider before you can operate razor-go cli on docker:
```
Comment (ashish10677) [inline README.md:23]: Just saw this, `setconfig` isn't the correct command. It is `setConfig`. Can you please update this as well.

File: README.md [hunk 2]
```diff
@@ -29,18 +31,42 @@ You can now execute razor-go cli commands by running:
...
 ### Prerequisites
-* Golang 1.15 or later must be installed.
-* Latest stable version of node is required.
-* Silicon chip based Mac users must go for node 15.3.0+
-* `geth` and `abigen` should be installed. (Skip this step if you don't want to fetch the bindings and build from scratch)
+
+- Golang 1.15 or later must be installed.
+- Latest stable version of node is required.
+- Silicon chip based Mac users must go for node 15.3.0+
+- `geth` and `abigen` should be installed. (Skip this step if you don't want to fetch the bindings and build from scratch)
```
Comment (ashish10677) [inline README.md:60]: I think `solc` and `jq` are a pre requisite as well

File: README.md [hunk 3]
```diff
@@ -78,65 +106,77 @@ Example:
 ```
 $ ./razor import
 🔑 Private Key:
-Password: 
+Password:
 ```
 
-__Before staking on Razor Network, please ensure your account has eth and RAZOR. For testnet RAZOR, please contact us on Discord.
+\_\_Before staking on Razor Network, please ensure your account has eth and RAZOR. For testnet RAZOR, please contact us on Discord.
```
Comment (ashish10677) [inline README.md:112]: Why 2 underscores here? The goal is to make this text italics.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

**Review by ashish10677 — APPROVED** [review]
LGTM

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  razor-go:
+    #platform: linux/x86_64 #Uncomment for silicon based MAC
```
Comment (SkandaBhat) [inline docker-compose.yml:4]: Remove this and add platform in readme

**Pull request merged**